subsystems/provisioning/definitions.go

```diff
@@ -370,6 +376,10 @@ type Config struct {
 
 	// If set, will explicitly enable or disable power save for all wifi connections managed by NetworkManager.
 	WifiPowerSave *bool `json:"wifi_power_save"`
+
+	// If set, will reboot the device after it has been offline for this duration
+	// 0 will disable this feature.
+	DeviceRebootAfterOfflineMinutes int `json:"device_reboot_after_offline_minutes"`
```

Comment:
Would prefer to see this be a `Timeout` instead of `int` as then the json gets parsed automatically for timeouts.

Reply:
We may need to change the default parsing to "minutes" for the timeout values (currently it assumes bare numbers are seconds.)